<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CubeDialog</class>
 <widget class="QDialog" name="CubeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose modeling type</string>
  </property>
  <widget class="QComboBox" name="cmbType">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dsbSize">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>88</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>88</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose the model type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Set the size of model(input units):</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
